gbpservice/neutron/nsf/core/cfg.py

```diff
@@ -0,0 +1,35 @@
+from oslo_config import cfg
+
+OPTS = [
+    cfg.IntOpt(
+        'workers',
+        default=1,
+        help=_('#of workers to create.')
+    ),
+    cfg.StrOpt(
+        'RpcLoadBalancer',
+        default='StickyRoundRobin',
+        choices=['RoundRobin', 'StickyRoundRobin'],
+        help=_('Check sc/core/lb.py for supported rpc lb algos')
+    ),
+    cfg.StrOpt(
+        'modules_dir',
+        default='gbpservice.neutron.nsf.core.test',
+        help=_('Modules path to import ')
+    ),
+    cfg.IntOpt(
+        'evs_polling_interval',
+        default=1,
+        help=_('Polling interval for events in seconds ')
```

Comment:
Can we add a little more help for all the parameters. Docs will be generated based on these help text. Lets make sure they make sense.